packages/p2p/src/components/my-ads/create-ad-form-payment-methods.jsx

```diff
@@ -78,24 +74,11 @@ const CreateAdFormPaymentMethods = ({ is_sell_advert, onSelectPaymentMethods })
     if (is_sell_advert) {
         if (my_profile_store.advertiser_has_payment_methods) {
             return (
-                <>
-                    {my_profile_store.advertiser_payment_methods_list.map((payment_method, key) => (
-                        <PaymentMethodCard
-                            is_vertical_ellipsis_visible={false}
-                            key={key}
-                            medium
-                            onClick={() => onClickPaymentMethodCard(payment_method)}
-                            payment_method={payment_method}
-                            style={selected_sell_methods.includes(payment_method.ID) ? style : {}}
-                        />
-                    ))}
-                    <PaymentMethodCard
-                        is_add={true}
-                        label={localize('Payment method')}
-                        medium
-                        onClickAdd={() => my_ads_store.setShouldShowAddPaymentMethodModal(true)}
-                    />
-                </>
+                <AdFormPaymentMethodsList
```

Comment:
Since this component is only used for Sell adverts, can we rename the component to `SellAdFormPaymentMethodsList` for clarity